src/main/java/teammates/storage/entity/Account.java

```diff
@@ -152,10 +145,6 @@ public void setStudentProfile(StudentProfile studentProfile) {
             return;
         }
         setIsStudentProfileEnabled(true);
-        if (studentProfile.getGoogleId().isEmpty()) { // only in local attribute tests
```

Comment:
Just realised while working on another PR that the javadoc for this method was not updated.